deploy/compliance-operator-chart/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
I guess it depends on how much work there is, but if we're going to have a "generator" of the Helm charts and the actual chart, it would be nice to have them separate.